<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>814</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Server-Side Application</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pbClose">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>350</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Close</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbSendMessage">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>290</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Send Messages</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbDisconnect">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>150</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Disconnect</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="teMessageHistory">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>431</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbServer">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Message History</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leTextMessage">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>401</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Hello2</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbShow">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Message</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbClearText">
    <property name="geometry">
     <rect>
      <x>322</x>
      <y>220</y>
      <width>131</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Messages</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leHostIPAddress">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>60</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>127.0.0.1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leHostPort">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>100</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>8000</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbServerIP">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>60</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Server IP Address</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbServerPort">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>100</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Server Port</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>814</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
